src/develop/blend_gui.c

```diff
@@ -1240,7 +1288,7 @@ gboolean blend_color_picker_apply(dt_iop_module_t *module, GtkWidget *picker, dt
     float picker_mean[8], picker_min[8], picker_max[8];
     float picker_values[4];
 
-    int in_out = (dt_key_modifier_state() == GDK_CONTROL_MASK) ? 1 : 0;
+    int in_out = ((dt_key_modifier_state() == GDK_CONTROL_MASK) && data->output_channels_shown) ? 1 : 0;
```

Comment:
I don't see all the code in this diff, but this may be also a gboolean.

Reply:
The `in_out` variable is used to distinguish between the input parametric mask widgets and output parametric mask widgets. It is also used as an index in the array (of two elements) storing those widgets.
Therefore, I prefer to keep it as an `int`.